conformance/reports/v1.1.0/kong-kubernetes-ingress-controller/README.md

```diff
@@ -0,0 +1,42 @@
+# Kong Kubernetes Ingress Controller
+
+## Table of Contents
+
+| API channel | Implementation version                                                              | Mode | Report                       |
+|-------------|-------------------------------------------------------------------------------------|------|------------------------------|
+| x           | [v3.2.0](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/v3.2.0) | x    | [link](./v3.2.0-report.yaml) |
```

Comment:
We need to rename the report according to the rules stated here: https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/README.md

Reply:
👍 Done.